Maple2Storage/Enums/StatAttribute.cs

```diff
@@ -88,121 +88,135 @@ public enum StatAttribute : short
     DropRate = 11048,
     QuestExp = 11049,
     QuestMeso = 11050,
-    InvokeEffect1 = 11051, // needs better name
-    InvokeEffect2 = 11052, // needs better name
-    InvokeEffect3 = 11053, // needs better name
-    PvPDamage = 11054,
-    PvPDefense = 11055,
-    GuildExp = 11056,
-    GuildCoin = 11057,
-    McKayXpOrb = 11058, //mc-kay experience orb value bonus
-    FishingExp = 11059,
-    ArcadeExp = 11060,
-    PerformanceExp = 11061,
-    AssistantMood = 11062, //assistant mood improvement rate
-    AssistantDiscount = 11063,
-    BlackMarketReduce = 11064,
-    EnchantCatalystDiscount = 11065,
-    MeretReviveFee = 11066,
-    MiningBonus = 11067,
-    RanchingBonus = 11068,
-    SmithingExp = 11069,
-    HandicraftMastery = 11070,
-    ForagingBonus = 11071,
-    FarmingBonus = 11072,
-    AlchemyMastery = 11073,
-    CookingMastery = 11074,
-    ForagingExp = 11075,
-    CraftingExp = 11076,
+    FishingExp = 11051,
+    ArcadeExp = 11052,
+    PerformanceExp = 11053,
+    InvokeEffect1 = 11054, // needs better name
+    InvokeEffect2 = 11055, // needs better name
+    InvokeEffect3 = 11056, // needs better name
+    PvPDamage = 11057,
+    PvPDefense = 11058,
+    GuildExp = 11059,
+    GuildCoin = 11060,
+    McKayXpOrb = 11061, //mc-kay experience orb value bonus
+    BlackMarketReduce = 11062,
+    EnchantCatalystDiscount = 11063,
+    MeretReviveFee = 11064,
+    MiningBonus = 11065,
+    RanchingBonus = 11066,
+    SmithingExp = 11067,
+    HandicraftMastery = 11068,
+    ForagingBonus = 11069,
+    FarmingBonus = 11070,
+    AlchemyMastery = 11071,
+    CookingMastery = 11072,
+    ForagingExp = 11073,
 
```

Comment:
My biggest concern of all these value changes is if this has been tested on equips. Has it? I'm almost positive  these values correspond to item stat attributes. If skills/effects have some different offset, perhaps we make some converter for this.